salt/modules/network.py

```diff
@@ -945,6 +945,21 @@ def get_hostname():
     return gethostname()
 
 
+def get_fqdn():
+    '''
+    Get fully qualified domain name
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' network.get_fqdn
+    '''
+
+    from socket import getfqdn
```

Comment:
We already have `socket` imported so there is no need for this line. You can remove it and change the line below to `return socket.getfqdn()`.